### PR TITLE
sbt 1.x compatible

### PR DESCRIPTION
**Pull request by magnolia-k**

### CODE REVIEW

File: src/main/scala-sbt-1.0/Compat.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  val ClasspathUtilities = internal.inc.classpath.ClasspathUtilities
+
+  def compileModules(m: UpdateReport): Option[Seq[sbt.ModuleReport]] = {
+     m.configuration(Compile).map(_.modules)
+  }
+
+  def createProcess: com.earldouglas.xwp.Compat.Process = {
+    new com.earldouglas.xwp.Compat.Process {
+      override def exitValue() = 0
+      override def destroy() {}
```
Comment (takezoe) [inline src/main/scala-sbt-1.0/Compat.scala]: Same as above.

File: src/main/scala-sbt-0.13/Compat.scala [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  val ClasspathUtilities = classpath.ClasspathUtilities
+
+  def compileModules(m: UpdateReport): Option[Seq[sbt.ModuleReport]] = {
+     m.configuration(Compile.name).map(_.modules)
+  }
+
+  def createProcess: com.earldouglas.xwp.Compat.Process = {
+    new com.earldouglas.xwp.Compat.Process {
+      override def exitValue() = 0
+      override def destroy() {}
```
Comment (takezoe) [inline src/main/scala-sbt-0.13/Compat.scala]: Avoid procedure syntax. It should be `destroy() = {}`.

**Comment by magnolia-k** [comment]
Thanks for comment!

I fixed it.

**Comment by takezoe** [comment]
@magnolia-k Thanks! Can we close #34?

**Comment by magnolia-k** [comment]
@takezoe I closed!

**Pull request merged**